02_Calculator_UI/04_Creating_variables.md

```diff
@@ -1,11 +1,15 @@
-If you see the text style for some of the **`Text`** widgets, you will note that we are repeating the code many times; the best practice is to make one variable that has the **`TextStyle`** object and use it inside our Text widgets multiple times.
+If you see the **`TextStyle`** for some of the **`Text`** widgets, you will note that you are repeating the same code multiple times, and repetitive code is the worst thing the programmer does, so the best practice is to make a variable that stores the **`TextStyle`** object and use it inside the Text widgets as many times as you want.
 
-This practice has two advantages:
+Why not repeat the code? Because of two main reasons:
 
-- Will make your code shorter.
-- When you want to change the size of text, you will change it one time in one place.
+- The code will get bigger as the project gets bigger.
+- A big and unorganized code requires more time and effort when it comes to fixing tiny or big bugs.
 
-1. Create a variable with the name `greenTextStyle` and give it the value of the styling that you are going to reuse for most of the buttons. To do that, write the following:
+Storing the repetitive code in a variable allows you to clean, organize, and understand the code when you get back to it after a long time, because God knows what you had written.
```

Comment:
@ZhraaBaharAluloom 
Replace God knows what you had written with better explanation.
Example: because with time you will forget what you wrote and it will not make sense to you.